app/views/admin/dashboard/dashboard.html.slim

```diff
@@ -37,7 +38,8 @@
         .person
           h3 = person.name
           .recent-projects
-            = render "recent_projects_grid_definition", grid: @people_grids[person]
+            = render partial: "recent_projects_grid_definition",
+                     locals: {person: person, grid: @people_grids[person]}
```

Comment:
oooh...that's the way RubyMine formatted the string when I broke it
I can break it another way tho